.github/workflows/pull-request.yml

```diff
@@ -0,0 +1,94 @@
+name: PR Workflow
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  linting_and_type-checking:
+    name: Linting, Formatting and Type checking
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        
+      - name: Get yarn cache directory path
+        id: yarn-cache-dir-path
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
+    
+      - name: Cache Node dependencies
+        uses: actions/cache@v3
```

Comment:
Upgrade the version to `@v4` which support node 20.

![Screenshot from 2024-03-05 02-32-19](https://github.com/json-schema-org/website/assets/66772290/710df414-1ddd-40fb-a3c6-fcc91b63ad74)